File: CsConsolaJuegito/CsConsolaJuegito/PuntajePartidasGuardar.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.IO;

namespace CsConsolaJuegito
{
    class PuntajePartidasGuardar
    {
    

        public void Guardar(bool x)
         {
            try
            {
                File.AppendAllText("C:/Users/oscar/Desktop/Puntaje/puntaje.txt", x + " ");
            }
            catch (Exception e)
            {
                Console.WriteLine("hubo un error: " + e);
            }
           
        }

      

    }
}

File: VbConsolaJuego/VbConsolaJuego/PuntajePartidasGuardar.vb
Imports System.IO

Public Class PuntajePartidasGuardar


    Sub Guardar(x As Boolean)

        Dim z As String
        z = CStr(x) + " "


        File.AppendAllText("C:/Users/oscar/Desktop/Puntaje/puntaje.txt", z)




    End Sub









End Class
